### DISCUSSION TITLE
Emacs Package

### DISCUSSION THREAD

**Discussion by CrownCrafter** [community/community#7013] [Copilot Conversations]
Please release a Emacs Package.

**Comment by VitalyAnkh** [comment]
Is GitHub working on it? GitHub Copilot will charge users a few time later, please support Emacs to improve the user experience of Emacs users.

**Comment by milanglacier** [comment]
I second on this. Please add support for emacs.

**Comment by rwilson-lib** [comment]
Please show Emacs users some love. Microsoft please add Emacs I am waiting of an official emacs support before subscribing. 
Neovim is great for coding by I can do a lot more with Emacs.

**Comment by robert-zaremba** [comment]
Any updates? Would be great to have an official Emacs plugin.

**Comment by Remillard** [comment]
I require a GitHub supported/certified Emacs client for use with Enterprise Copilot.  I'm not the only one in the organization who desires this, we've unofficially have a user's group swapping Emacs knowledge around.  However until GitHub officially endorses/supports/certifies a package for Emacs, we cannot use the tool.
